packages/lesswrong/components/localGroups/LocalGroupPage.jsx

```diff
@@ -81,6 +81,7 @@ class LocalGroupPage extends Component {
     const { classes } = this.props;
     const { groupId } = this.props.params;
     const group = this.props.document;
+    const htmlBody = group ? {__html: group.htmlBody} : {_html: ""};
```

Comment:
This is the only place where you provide a fallback option if the group doesn't exist – curious if there are particular reasons either this one has it or the other two don't?

Reply:
For this one it threw me an error during loading of the page. So that's mostly it.